cli/cli.go

```diff
@@ -104,57 +99,34 @@ func createCliCommandTree(cmd *cobra.Command) {
+	cmd.PersistentFlags().String("log-format", "text", tr("The output format for the logs, can be: %s", strings.Join(validLogFormats, ", ")))
 	cmd.RegisterFlagCompletionFunc("log-level", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return validLogLevels, cobra.ShellCompDirectiveDefault
 	})
-	cmd.PersistentFlags().String("log-file", "", tr("Path to the file where logs will be written."))
-	validLogFormats := []string{"text", "json"}
-	cmd.PersistentFlags().String("log-format", "", tr("The output format for the logs, can be: %s", strings.Join(validLogFormats, ", ")))
 	cmd.RegisterFlagCompletionFunc("log-format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return validLogFormats, cobra.ShellCompDirectiveDefault
 	})
 	validOutputFormats := []string{"text", "json", "jsonmini", "yaml"}
-	cmd.PersistentFlags().StringVar(&outputFormat, "format", "text", tr("The output format for the logs, can be: %s", strings.Join(validOutputFormats, ", ")))
+	cmd.PersistentFlags().BoolP("verbose", "v", false, tr("Print the logs on the standard output."))
+	cmd.PersistentFlags().String("format", "text", tr("The output format for the logs, can be: %s", strings.Join(validOutputFormats, ", ")))
+	cmd.PersistentFlags().Bool("no-color", false, "Disable colored output.")
 	cmd.RegisterFlagCompletionFunc("format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return validOutputFormats, cobra.ShellCompDirectiveDefault
 	})
 	cmd.PersistentFlags().StringVar(&configFile, "config-file", "", tr("The custom config file (if not specified the default will be used)."))
 	cmd.PersistentFlags().StringSlice("additional-urls", []string{}, tr("Comma-separated list of additional URLs for the Boards Manager."))
-	cmd.PersistentFlags().Bool("no-color", false, "Disable colored output.")
 	configuration.BindFlags(cmd, configuration.Settings)
 }
 
-// convert the string passed to the `--log-level` option to the corresponding
-// logrus formal level.
-func toLogLevel(s string) (t logrus.Level, found bool) {
-	t, found = map[string]logrus.Level{
-		"trace": logrus.TraceLevel,
-		"debug": logrus.DebugLevel,
-		"info":  logrus.InfoLevel,
-		"warn":  logrus.WarnLevel,
-		"error": logrus.ErrorLevel,
-		"fatal": logrus.FatalLevel,
-		"panic": logrus.PanicLevel,
-	}[s]
-
-	return
-}
-
-func parseFormatString(arg string) (feedback.OutputFormat, bool) {
-	f, found := map[string]feedback.OutputFormat{
-		"json":     feedback.JSON,
-		"jsonmini": feedback.JSONMini,
-		"text":     feedback.Text,
-		"yaml":     feedback.YAML,
-	}[strings.ToLower(arg)]
-
-	return f, found
-}
-
 func preRun(cmd *cobra.Command, args []string) {
+	if cmd.Name() == "daemon" {
+		return
+	}
+
 	configFile := configuration.Settings.ConfigFileUsed()
 
 	// initialize inventory
```

Comment:
If we are removing telemetry we should remove also this one?

Reply:
I think it would be better to remove it in another PR, this one has already too many changes.